src/format/tests.rs

```diff
@@ -2571,6 +2606,8 @@ pub fn two() {
 }
 ",
     );
+
+    // Bitstrings
```

Comment:
Leftover comment ? :)

Reply:
I need to move the new asserts down here, I've got them in a separate test at the moment because it makes them easier to run on their own. At some point I'd like to try and split these mega-tests up into smaller, better documented ones, but that's a task for another day.

Reply:
Makes sense. I'd be happy to have them as a different test function if you'd like to split the others out too at a later date.

Reply:
I'll move the format ones back into `module_test()` because there's only 2 asserts anyway. The typ ones I'll leave in their own test as there's quite a few.